doc/dbus/org.opensuse.Agama.Questions1.doc.xml

```diff
@@ -62,28 +64,30 @@ when the question is answered and the answer is successfully read.
       @data: additional data that is specific to given question class. Can be
              used for additional details about question that helps with UI
              or with matching automatic answers.
-      @result: Object path of created question.
 
       Creates new question with generic base and additional WithPassword interface.
+      Returns the object path of the created question.
     -->
     <method name="NewWithPassword">
       <arg name="class" direction="in" type="s"/>
       <arg name="text" direction="in" type="s"/>
       <arg name="options" direction="in" type="as"/>
       <arg name="default_option" direction="in" type="s"/>
       <arg name="data" direction="in" type="a{ss}"/>
-      <arg name="result" direction="out" type="o"/>
+      <arg direction="out" type="o"/>
     </method>
     <method name="Delete">
       <arg name="question" direction="in" type="o"/>
     </method>
+
     <!--
       UseDefaultAnswer:
 
       Switches questions to be automatically answered by default answer.
       After this method call each follow up question is immediatelly answered.
       Useful for doing non interactive installation.
     -->
-    <method name="UseDefaultAnswer" />
+    <method name="UseDefaultAnswer">
+    </method>
```

Comment:
I am just curious why this change is needed? It is due to output of introspect method?

Reply:
Yes, introspection has the whitespace text node. We know it can be ignored because we know the schema/DTD but the checker is too simple for that.